setup.py

```diff
@@ -10,5 +10,5 @@
     "distributed computing framework Ray.",
     url="https://github.com/ray-project/xgboost_ray",
     install_requires=[
-        "xgboost>=1.3", "ray", "numpy>=1.16,<1.20", "pandas", "pyarrow"
+        "xgboost", "ray", "numpy>=1.16,<1.20", "pandas", "pyarrow"
```

Comment:
>=0.90?

Reply:
I guess we could add `>=0.90` - this branch is still compatible with xgboost >= 1.3 (and probably with any version between those, but I haven't tried)

Reply:
yeah that'd be great